src/SFA.DAS.ApprenticeFeedback.Data/Repository/ApprenticeFeedbackRepository.cs

```diff
@@ -59,10 +60,23 @@ public async Task<ApprenticeFeedbackTarget> GetApprenticeFeedbackTarget(Guid app
         => await _dbContext.ApprenticeFeedbackTargets.
             FirstOrDefaultAsync(aft => aft.ApprenticeId == apprenticeId && aft.ApprenticeshipId == commitmentsApprenticeshipid);
 
+        public async Task<ApprenticeFeedbackTarget> GetApprenticeFeedbackTargetById(Guid apprenticeId)
+        => await _dbContext.ApprenticeFeedbackTargets.
+            FirstOrDefaultAsync(aft => aft.ApprenticeId == apprenticeId);
+
 
-        public async Task<List<Domain.Entities.Attribute>> GetProviderAttributes()
+        public async Task<IEnumerable<Domain.Entities.Attribute>> GetAttributes()
         {
-            return await _dbContext.Attributes.ToListAsync();
+            return await _dbContext.Attributes.AsNoTracking().ToListAsync();
```

Comment:
AsNoTracking has it's reasons, was this added as part of an investigation or was there a conscious decision?

Reply:
Added as part of the investigation, I will take that out